tt_metal/Glossary.md

```diff
@@ -0,0 +1,145 @@
+# Glossary of Terms
+## **Tenstorrent Devices:** ##
+- **Blackhole -** Tenstorrent's next generation AI compute solution. 140 Tensix Cores.
+- **Galaxy -** Tenstorrent's high density rack-mounted AI compute solution, intended for corporations.
+- **Grayskull -** Tenstorrent's entry level DevKit. 120 Tensix Cores.
+- **LoudBox -** Tenstorrent's desktop workstation equipped with Tenstorrent hardware and software.
+- **Wormhole -** Tenstorrent’s GPU device. 80 Tensix Cores.
+## **Tenstorrent Software:** ##
+- **Metal Trace -** Performance optimization feature.
+- **Runtime -** Bottom of the Metalium software stack. Controls kernel execution and collects results from kernel executions.
+- **Tracy -** Tenstorrent's profiling tool.
+- **TT-Buda -** A compute framework used to develop, run, and analyze ML workloads on Tenstorrent hardware.
+- **TT-Fabric -**
+- **TT-Forge -** Tenstorrent’s graph compiler.
+- **TT-Metalium -** Tenstorrent’s low-level programming model.
+- **TT-MLIR -** Tenstorrent’s MLIR-based compiler.
+- **TT-NN -** Tenstorrent’s library of optimized operators.
+- **Watcher -** Tenstorrent monitoring tool.
+## **Acronyms:** ##
```

Comment:
I would group them into meaningful sections. Right now, this list seems really random.